projects/aca-content/src/lib/components/search/search-input/search-input.component.scss

```diff
@@ -44,13 +40,15 @@ $top-margin: 12px;
   }
 }
 
-.app-search-options-menu {
+/* stylelint-disable */
```

Comment:
I think it's better to use `stylelint-disable-next-line` with the rule we want to disable not to block the other checks for this part